ShowAnchorCloud.glyphsReporter/Contents/Resources/plugin.py

```diff
@@ -109,3 +109,6 @@ def background(self, layer):
         except:
             print("Oops!", sys.exc_info()[0], "occured.")
             traceback.print_exc(file=sys.stdout)
+
+    def shouldDrawAccentCloudForLayer_(self, layer):
```

Comment:
I don't understand how this works. Is this method supposed to be called from somewhere?

Reply:
I don’t know either, I stole it from https://github.com/mekkablue/ShowMarkPreview/blob/c248dc4dc80b136a10af3b3c1615666e95902278/ShowMarkPreview.glyphsReporter/Contents/Resources/plugin.py#L228 and experimented with it and it does work.

Reply:
I guessed that Glyphs will call this method with each shown layer to decide whether it should draw its own anchor cloud or not.